velox/scripts/benchmark_velox.sh

```diff
@@ -30,6 +32,7 @@ Benchmark Options:
   -q, --queries "1 2 ..."                 Query numbers to run, specified as a space-separated list of query numbers (default: all queries for benchmark type)
   -d, --device-type "cpu gpu"             Devices to test: cpu, gpu, or "cpu gpu" (default: "cpu gpu")  
   -p, --profile BOOL                      Enable profiling: true or false (default: false)
+  --docker-runtime MODE                   Docker runtime mode: cpu|gpu|auto (default: auto)
```

Comment:
Why do we need this as a separate flag from `device-type`? Also, what is the use case for the `auto` option?

Reply:
>device-type is specific to running benchmarks where as this is specific to the docker runtime

In what case would we want a docker runtime that does not match the specified `device-type`?

>auto option automatically detects the runtime on the fly by checking nvidia-smi command and set the DOCKER_RUNTIME var based upon it's results.,

I understand what it does. My question is where the auto-detection requirement is coming from. When running these benchmarks, we typically know what device type we are targeting. This appears to add complexity that may not be needed.